config.md

```diff
@@ -104,10 +104,10 @@
 |credentials|CORS setting to control whether a browser allows credentials to be sent to this API|`boolean`|`true`
 |debug|Whether debug is enabled for the CORS implementation|`boolean`|`false`
 |enabled|Whether CORS is enabled|`boolean`|`true`
-|headers|CORS setting to control the allowed headers|`string`|`[*]`
+|headers|CORS setting to control the allowed headers|`[]string`|`[*]`
 |maxAge|The maximum age a browser should rely on CORS checks|[`time.Duration`](https://pkg.go.dev/time#Duration)|`600`
-|methods| CORS setting to control the allowed methods|`string`|`[GET POST PUT PATCH DELETE]`
-|origins|CORS setting to control the allowed origins|`string`|`[*]`
+|methods| CORS setting to control the allowed methods|`[]string`|`[GET POST PUT PATCH DELETE]`
```

Comment:
This one is here: https://github.com/hyperledger/firefly-common/blob/5a2539a3727b6eeecee07da0297198f7bcbd2bed/pkg/i18n/en_base_config_descriptions.go#L106